daemon/api.go

```diff
@@ -172,6 +174,12 @@ var (
 		UserOK: false,
 		POST:   postCreateUser,
 	}
+
+	deviceIdentityCmd = &Command{
+		Path:     "/v2/experimental-device-identity",
```

Comment:
It's not clear that this should be an endpoint on its own. It should probably be a DeviceManager in the state instead, as this isn't something the user or an external application should trigger, and it's something we'll potentially need to iterate on in case of failures, etc. It will also need to iterate with functionality which may be multi-step, such as hooks, so it'll need to integrate with the task and change system.